iroha_schema/iroha_schema_bin/src/main.rs

```diff
@@ -4,7 +4,7 @@
 
 use std::collections::BTreeMap;
 
-use iroha_data_model::{prelude::*, query::QueryBox};
+use iroha_data_model::{prelude::*};
```

Comment:
Styling!

Reply:
Yes, the formatter caught it

Reply:
Made a comment to make sure that it wont be merged till it is fixed :)

Reply:
fixed